pkg/catalog/utils/utils.go

```diff
@@ -79,3 +83,17 @@ func Contains(collection []string, key string) bool {
 func GetCatalogImageCacheName(catalogName string) string {
 	return fmt.Sprintf("%s-catalog-image-list", catalogName)
 }
+
+func GetCatalogChartPath(catalog *v3.Catalog, bundledMode bool) (string, error) {
+	if bundledMode {
+		switch catalog.Name {
+		case "helm3-library", "library", "system-library":
+			return filepath.Join(helmlib.InternalCatalog, catalog.Name), nil
+		case "rancher-charts", "rancher-partner-charts", "rancher-rke2-charts":
+			return filepath.Join(helmlib.InternalCatalog, "v2", catalog.Name), nil
+		default:
+			return "", fmt.Errorf("cannot find bundled catalog chart path for catalog %s", catalog.Name)
+		}
```

Comment:
I didn't need to add all these switch cases, but I did for good measure in case someone wants to use this helper function in the future 😄